app-front/src/config/index.ts

```
@@ -0,0 +1,64 @@
+    .reduce((acc, [key]) => {
+      const k = key.replace("NEXT_", "").toLowerCase();
+      acc[k] = process.env[key];
+      return acc;
+    }, {} as Record<string, ConfigObject>);
+}*/
+
+function objectToMap(obj: ConfigObject): Map<string, ConfigObject> {
+  const map = new Map();
+  if (typeof obj === "object" && obj !== null) {
+    for (const key in obj) {
+      if (obj.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)